metadata_templates.py

```diff
@@ -33,19 +34,23 @@ def mt_build_json_input(entity_name, entity_type, operation, value, units):
 def mt_attach(callback, entity_name, entity_type, value, units):
```

Comment:
Add brief descriptions for the `mt_*` functions. The descriptions do not need to be exhaustive since the functions are private to the implementation.

Reply:
done